WolvenKit/Forms/frmMain.cs

```diff
@@ -818,7 +818,7 @@ public frmCR2WDocument LoadDocument(string filename, MemoryStream memoryStream =
                     {
                         var rigDoc = new frmCR2WDocument();
                         var basePath = doc.File.FileName.Split(new string[] { "characters" }, StringSplitOptions.None)[0];
-                        var modelName = Path.GetFileName(doc.File.FileName).Split('_', '.')[3];
+                        var modelName = Path.GetFileName(doc.File.FileName).Split('_', '.');
```

Comment:
If you do that modelname will be an array not a string and you need a string here:
```
 var rigPath = $@"{basePath}characters\base_entities\{modelName}_base\{modelName}_base.w2rig";
```